### ISSUE TITLE
Hot Water not supported

### ISSUE THREAD

**Issue by lwsrbrts** [lwsrbrts/PoSHive#6]
PoSHive does not support Hive heating & hot water systems (dual channel receiver).

**Comment by lwsrbrts** [comment]
Talking to myself but this is a known issue and I can't fix it unless I get access to a Hive heating & hot water system or I'm able to decipher a method for selecting the correct place to send my commands.

If you have a heating and hot water system, it would be great if you could run the following script and send me the `Nodes.json`output file which should be saved on your desktop.

``` powershell
Import-Module -Name PoSHive

$HiveUsername = 'user@domain.com'
$HivePassword = 'hivewebsitepassword'

$Hive = [Hive]::new($HiveUsername, $HivePassword)

$Hive.Login()

$Hive.Devices | ConvertTo-Json -Depth 99 | Out-File "$([environment]::GetFolderPath('Desktop'))\Devices.json"
$Hive.Products | ConvertTo-Json -Depth 99 | Out-File "$([environment]::GetFolderPath('Desktop'))\Products.json"

$Hive.Logout()
```

I'll try to use this to work out how I can support the heating and hot water system.

**Comment by lwsrbrts** [comment]
I have a family member who has just bought a heating and hot water system so I'll be able to look at and implement this soon (ish). It might take a couple of months yet but it'll come.

**Comment by lwsrbrts** [comment]
Now implemented in version 2.0.0 of the class.
